pkg/agent/openflow/pipeline.go

```diff
@@ -41,38 +41,141 @@ import (
+	//          IPv6Table        = newTable("IPv6", stageValidation, pipelineIP)
+	//       ```
+	//      * If you want to add a table called `FooTable` just before `ARPResponderTable` in pipelineARP, then the table
+	//        should be declared before `ARPResponderTable`:
+	//       ```go
+	//          FooTable          = newTable("Foo", stageOutput, binding.PipelineARP)
+	//          ARPResponderTable = newTable("ARPResponder", stageOutput, binding.PipelineARP)
+	//       ```
+	//       * If you want to add a table called `FooTable` just after `ConntrackStateTable` in pipelineARP, then the
+	//         table should be declared after `ConntrackStateTable`:
+	//       ```go
+	//          SNATConntrackTable  = newTable("SNATConntrackZone", stageConntrackState, pipelineIP)
+	//          ConntrackTable      = newTable("ConntrackZone", stageConntrackState, pipelineIP)
+	//          ConntrackStateTable = newTable("ConntrackState", stageConntrackState, pipelineIP)
+	//          FooTable            = newTable("Foo", stageConntrackState, pipelineIP)
+	//       ```
+	//  - Reference the new table in a feature in file pkg/agent/openflow/framework.go. The table can be referenced by multiple
+	//    features if multiple features need to install flows in the table. Note that, if the newly added table is not
+	//    referenced by any feature or the features referencing the table are all inactivated, then the table will not
+	//    be realized in OVS; if at least one feature referencing the table is activated, then the table will be realized
+	//    at the desired position in OVS pipeline.
+	//
+	// How to forward packet between tables with a proper action in FlexiblePipeline?
+	//
+	// |   table A   | |   table B   | |   table C   | |   table D   | |   table E   | |   table F   | |   table G   |
+	// |   stage S1  | |                           stage S2                          | |          stage S4           |
+	//
+	//  - Next is used to forward packets to the next table. E.g, A -> B, B -> C, C -> D, etc.
+	//  - GotoTable is used to forward packets to a specific table, and the target table ID should be greater than the
+	//    current table ID. Note that, it is recommended to use GotoTable to forward packets with a stage, e.g, B -> D,
+	//    C -> E. However, in future OVS pipeline development, if packets need to be forwarded to a table in another stage
+	//    directly anyway, e.g, A -> C, B -> G, GoTable can also be used.
+	//  - GotoStage is used to forward packets to a specific stage. Note that, packets are forwarded to the first table of
+	//    the target stage, and the first table ID of the target stage should be greater than the current table ID. E.g,
+	//    A -> S4 (F), D -> S4 (F) are fine, but D -> S1 (A), F -> S2 (B) are not allowed. It is recommended to use
+	//    GotoStage to forward packets across stages.
+	//  - ResubmitToTables is used to forward packets to one or multiple tables. E.g, B -> D E, E -> B. Note that, the
```

Comment:
E.g.

Reply:
Probably: "It should be used only when the target table ID is smaller than the current table ID, like E -> B; otherwise GotoTable should be used".

Reply:
We also need this to forward packets to multiple tables. How about this:
```
//  ResubmitToTables should be used when the target table ID is smaller than the current table ID, like E -> B;
//    otherwise GotoTable should be used. In additions, it is also used to forward packets to multiple tables, like
//    B - > D E.
```